playwright/_impl/_wait_helper.py

```diff
@@ -13,22 +13,27 @@
 # limitations under the License.
 
 import asyncio
+import uuid
```

Comment:
do we already have it as a dependency? I think it needs to be added to the setup.py.